libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html

```diff
@@ -3,6 +3,7 @@
         <div id="text-content" class="content">
             <h1 class="title">{{titleText}}</h1>
             <p class="paragraph" *ngFor="let paragraph of paragraphTexts">{{paragraph}}</p>
+            <ng-content select="[banner]"></ng-content>
         </div>
```

Comment:
The reusable button needs to be added here, not transclusion. This ensures the button is not re-added on every component that has a button, also this can't be transcluded because we'll only buttons here for cosmetic reasons.

Reply:
Check the issue description again